packages/zowe-explorer/README.md

```diff
@@ -104,81 +89,52 @@ The first time profiles are used you will be prompted for user name and password
+1. If only a global or project level config is in place, it opens to be edited. If both a global and project level config are in place, the user must select which file to edit.
    ![Edit Config Location Option](/docs/images/ZE-edit-options.png)
    <br /><br />
 
 ### Profile validation
 
-**Note:** The following information applies to Zowe CLI V1 profiles (one yaml file for each user profile) and Zowe CLI team profiles (Zowe CLI V2).
-
-Zowe Explorer includes the profile validation feature that helps to ensure that the specified connection to z/OS is successfully established and your profile is ready for use. If a profile is valid, the profile is active and can be used.
+Zowe Explorer includes the profile validation feature that helps to ensure that the specified connection to z/OS is successfully established and your profile is ready for use. If the API connection is valid, the profile is active and can be used.
 
 By default, this feature is automatically enabled. You can disable the feature by right-clicking on your profile and selecting the **Disable Validation for Profile** option. Alternatively, you can enable or disable the feature for all profiles in the VS Code settings
 
 1. In VS Code, navigate to **Settings**.
-2. Navigate to Zowe Explorer settings.
-3. Check the **Automatic Profile Validation** checkbox to enable the automatic validation of profiles option. Uncheck to disable.
-4. Restart VS Code.
+1. Navigate to Zowe Explorer settings.
+1. Check the **Automatic Profile Validation** checkbox to enable the automatic validation of profiles option. Uncheck to disable.
+1. Restart VS Code.
 
 ### Use base profile and token with existing profiles
 
 As a Zowe user, you can leverage the base profile functionality to access multiple services through Single Sign-on. Base profiles enable you to authenticate using the Zowe API Mediation Layer (API ML). You can use base profiles with more than one service profile. For more information, see [Base Profiles](https://docs.zowe.org/stable/user-guide/cli-using-using-profiles/#base-profiles).
 
-**Note:** Before using the base profile functionality with v1 profiles, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) v6.0.0 or higher installed.
-
-1. Zowe Explorer has a right click action for profiles to log in and log out of the authentication service for existing Base profiles. If a v1 Base profile hasn't been created, open a terminal and run the following Zowe CLI command: `zowe auth login apiml`.
-2. Follow the instructions to complete the login.
-   A local base profile is created that contains your token.
-
-   **Note:** For more information about the process, see [Token Management](https://docs.zowe.org/stable/user-guide/cli-using-integrating-apiml/#how-token-management-works).
+Zowe Explorer has a right click action for profiles to log in and log out of the authentication service for existing Base profiles.
```

Comment:
I don't think we capitalize "base" profiles anywhere in Zowe Docs. I'd leave it lowercase.